com.endlessm.apps.Sdk.json.in.in

```diff
@@ -501,6 +501,41 @@
                 "tar -C /usr/share/eos-hotdoc-theme -xJf eos-hotdoc-theme-0.0.0.tar.xz"
             ]
         },
+        {
+            "name": "bison",
```

Comment:
This leaves `bison` and `flex` in the SDK, which is not something I'm really looking forward to.

In any case, though, both `bison` and `flex` are in the org.freedesktop.Sdk run time, so they should not be needed.

Reply:
Fair enough. As long as mustache-c builds, I'm happy.